Add rfc 0002 governance and control of the e bgf (#48)
internet-sicherheit/eco-blockchain-governance: 0002-governance-and-control-of-the-eBGF/README.md

# 0002: Governance and control of the eBGF
- Authors: [David Maas](david_maas@hotmail.de)
- Status: [PROPOSED](/README.md#proposed)
- Since: 2020-09-09
- Status Note: initial version  
- Supersedes:
- Start Date: 2020-09-09
- Tags: rfc, control, ebgf

## Summary

> In the following it will be explained how a Pull Request for a new RFC is regulated and who is responsible for each working group. To make this process as transparent and effective as possible. 

Reviewer: ```suggestion
This RFC explains how a PR for a new RFC is processed and who is responsible for each working group, in order to make this process as transparent and effective as possible. 
```

## Motivation

> Clearly defined rules make working on specific topics much easier. Through these rules, the work process is sufficiently described and no questions should remain open. In addition, the respective contact person for the topics is directly known and can be contacted directly if necessary.

Reviewer: ```suggestion
Clearly defined rules make working on specific topics much easier. 
Through these rules, the work process is sufficiently described and no questions should remain open. 
In addition, the respective contact person for the topics is directly known and can be contacted directly if necessary.
```

## Process

We try to conduct reviews of the set PRs at fixed times during the day.
This way nobody is torn away from requested reviews during the actual work. 

Currently, there are the following review times:
  - The first thing in the morning
  - Right after lunch

> 

Reviewer: I think this would not be true 😄 
Also, I don't think we would ever add or guarantee specific working times.

Reviewer 2: Maybe like broad work like:
- Reviews and suggestions are reviewed once every two or three weeks?

Reviewer: "Should occur at least once every x weeks" or something.

Author: Have adapted this again based on your suggestions for the wording.

> The set PRs must be reviewed by 3 maintainers. The maintainers of the working group is mandatory, but a delegate must be appointed if he is unable to attend. If the PR is subsequently verified, it can be merged into the master.

Reviewer: ```suggestion
The set PRs must be reviewed by at least 2 maintainers. 
One specific maintainer of the working group is mandatory, but a delegate must be appointed if he is unable to attend. 
If the PR is subsequently approved, it can be merged into master by one of the maintainers.
```

Reviewer 2: What happens if one or both are unable to perform their duties. Say Corona? or death worst case scenario. How does this legacy continue for years.

Reviewer: We should add this as a open point, I assume there should be a word for this, but I don't know, so let's call it: Long term maintenance and maintainer legacy 😆

> The merge is carried out by the maintainers of the working group. If the maintainer is currently unable to attend, a delegate will be appointed. 

Reviewer: ```suggestion
The merge is carried out by the maintainers of the working group using **Squash and merge** and editing the commit message accordingly.
When editing the commit message, please try to keep co-authors in the commit message, so GitHub can appropriately attach authors to the commit. 
If a maintainer is currently unable to attend to this task, a delegate will be appointed. 
```
